coderd/coderd.go

```diff
@@ -1239,10 +1239,18 @@ func (api *API) CreateInMemoryProvisionerDaemon(dialCtx context.Context, name st
 		}
 	}()
 
+	// All in memory provisioners will be apart of the default org for now.
+	//nolint:gocritic // in-memory provisioners are owned by system
+	defaultOrg, err := api.Database.GetDefaultOrganization(dbauthz.AsSystemRestricted(dialCtx))
```

Comment:
I think you've already noticed that this will fail if the default organization hasn't been created.  So think we need have a migration that creates a default org if none exists before we can have this.

Reply:
Yea, I will make sure this PR is merged first: https://github.com/coder/coder/pull/12412/files